src/hotspot/share/gc/g1/g1PageBasedVirtualSpace.cpp

```diff
@@ -101,7 +101,7 @@ size_t G1PageBasedVirtualSpace::uncommitted_size()  const {
 }
 
 size_t G1PageBasedVirtualSpace::addr_to_page_index(char* addr) const {
-  return (addr - _low_boundary) / _page_size;
+  return pointer_delta(addr, _low_boundary, _page_size);
 }
```

Comment:
I think the method is unused and can be removed (couldn't find it being referenced using 'grep')